src/renderer/src/assets/main.css

```diff
@@ -169,3 +169,115 @@ code {
     display: none;
   }
 }
+
+/**
+ * Account Management Styles
+ * @dev Core styles for account listing and interaction components
+ * @notice Uses flexbox for layout and responsive design
+ */
+
+/**
+ * Account List Container
+ * @dev Vertical flex container for account items
+ * @notice Consistent spacing between items with gap
+ */
+.account-list {
```

Comment:
generally, i would avoid using this over the in-component style because that will scope your css classes. look for lang="postcss"